server/sandbox_stop_test.go

```diff
@@ -36,6 +36,8 @@ var _ = t.Describe("PodSandboxStatus", func() {
 					gomock.Any(), gomock.Any()).Return(nil),
 				ociRuntimeMock.EXPECT().WaitContainerStateStopped(gomock.Any(),
 					gomock.Any()).Return(nil),
+				ociRuntimeMock.EXPECT().UpdateContainerStatus(gomock.Any()).
+					Return(nil),
```

Comment:
I guess this insertion has to be moved after the 
```
runtimeServerMock.EXPECT().StopContainer(gomock.Any()).
					Return(nil),
```
below. :point_down:

Reply:
yup, done